tests/base_d_asa/test_open_account.py

```diff
@@ -23,32 +28,26 @@ def test_pass_open_account(
     holding = algorand.account.random()
     payment = algorand.account.random()
 
-    base_d_asa_client_empty.open_account(
-        holding_address=holding.address,
-        payment_address=payment.address,
-        transaction_parameters=OnCompleteCallParameters(
+    base_d_asa_client_empty.send.open_account(
+        args=OpenAccountArgs(
+            holding_address=holding.address,
+            payment_address=payment.address,
+        ),
+        params=CommonAppCallParams(
+            sender=account_manager.address,
             signer=account_manager.signer,
-            boxes=[
-                (base_d_asa_client_empty.app_id, account_manager.box_id),
-                (
-                    base_d_asa_client_empty.app_id,
-                    DAsaAccount.box_id_from_address(holding.address),
-                ),
-            ],
+            static_fee=AlgoAmount.from_algo(1),
```

Comment:
Why is this needed?

Reply:
Ill keep this unresolved until i do a few more tests, ignore this for now ill add explanation later